cryptographic_estimators/SDFqEstimator/sdfq_helper.py

```diff
@@ -138,40 +118,35 @@ def _optimize_m4ri(n: int, k: int, mem=inf):
 
 
 def _mem_matrix(n: int, k: int, r: int):
-    """
-    Memory usage of parity check matrix in vector space elements
-
-    INPUT:
-
-    - ``n`` -- length of the code
-    - ``k`` -- dimension of the code
-    - ``r`` -- block size of M4RI procedure
+    """Calculates the memory usage of the parity check matrix in vector space elements.
 
-    EXAMPLES::
+    Args:
+        n (int): The length of the code.
+        k (int): The dimension of the code.
+        r (int): The block size of the M4RI procedure.
 
-        sage: from cryptographic_estimators.SDEstimator import _mem_matrix
-        sage: _mem_matrix(n=100,k=20,r=0) # random
+    Returns:
+        The memory usage of the parity check matrix in vector space elements.
 
+    Examples:
+        >>> from cryptographic_estimators.SDEstimator import _mem_matrix
+        >>> _mem_matrix(n=100,k=20,r=0) # random output # doctest: +SKIP
     """
     return n - k + 2 ** r
 
 
+# TODO: Remove?
 def _list_merge_complexity(L: float, l: int, hmap: bool):
-    """
-    TODO remove?
-    Complexity estimate of merging two lists exact
-
-    INPUT:
-
-    - ``L`` -- size of lists to be merged
-    - ``l`` -- amount of bits used for matching
-    - ``hmap`` -- indicates if hashmap is being used (Default 0: no hashmap)
-
-    EXAMPLES::
+    """Complexity estimate of merging two lists.
 
-        sage: from cryptographic_estimators.SDEstimator import _list_merge_complexity
-        sage: _list_merge_complexity(L=2**16,l=16,hmap=1) # random
+    Args:
+        L (float): Size of lists to be merged.
+        l (int): Amount of bits used for matching.
+        hmap (bool): Indicates if a hashmap is being used (Default: False).
 
+    Examples:
+        >>> from cryptographic_estimators.SDEstimator import _list_merge_complexity
+        >>> _list_merge_complexity(L=2**16,l=16,hmap=1) # random output # doctest: +SKIP
```

Comment:
I have no idea, why we should remove this function or comment. Just remove the `TODO`